domdiv/cards.py

```diff
@@ -56,6 +56,17 @@ def __init__(self, name=None, cardset='', types=None, cost='', description='',
             self.count = []
         else:
             self.count = [int(count)]
+        self.randomizer = randomizer
+        if self.randomizer:
+            # Remove classes of cards that never have Randomizers
+            # Card types
+            if any(self.isType(t) for t in [
```

Comment:
Is there a strong reason to not just set these individually to `false` in the db files? I'd like to try to get rid of as many places as possible where cards receive special handling by type/name in addition to what's in the DB.

If there is a strong reason, I think we should at least make a new module (`constants.py`) that collects all of these lists of card names as well as other constants currently in use like the various option lists.

Reply:
I agree.  I'll move this into the db files.